OpenOversight/app/main/forms.py

```diff
@@ -165,8 +166,8 @@ class AddOfficerForm(Form):
     star_no = StringField('Badge Number', default='', validators=[
         Regexp('\w*'), Length(max=50)])
     unique_internal_identifier = StringField('Unique Internal Identifier', default='', validators=[Regexp('\w*'), Length(max=50)])
-    rank = SelectField('Rank', default='PO', choices=RANK_CHOICES,
-                       validators=[AnyOf(allowed_values(RANK_CHOICES))])
+    rank = StringField('rank', default='', validators=[
```

Comment:
I think there is a danger in making these string fields instead of selecting from the database just because based on the error rates we've seen with people adding data to the openoversight.com database, we'll end up with a bunch of mispellings/whitespace all for the same officer rank. I think we need to provide some more guard rails here for people doing data entry. How about we make these rank selections as `QuerySelectField` for `FindOfficerForm` and `AssignmentForm` and then have a "Add/edit/delete rank" page.

Reply:
Good call, we don't want to end up with a bunch of typos. @camille-s is working on normalizing our rank data since our initial rosters were pretty inconsistent, so we'll be able to create the rank choices from that.

Reply:
For Baltimore we have like 15 sworn officer ranks and then like 80 other job titles. That seems like too many to display in a form on the Add/Edit Department page. Does it seem okay to just include the sworn officer job titles on the Add/Edit Department page and assume admins will import the other job titles via the `bulk-add-officers` import command?

Reply:
Sure that's OK with me, we can file a followup for adding editing of non sworn officer roles via the web interface